ddl/ddl_api.go

```diff
@@ -500,17 +512,17 @@ func checkGeneratedColumn(colDefs []*ast.ColumnDef) error {
 	return nil
 }
 
-func checkTooLongColumn(colDefs []*ast.ColumnDef) error {
-	for _, colDef := range colDefs {
-		if len(colDef.Name.Name.O) > mysql.MaxColumnNameLength {
-			return ErrTooLongIdent.Gen("too long column %s", colDef.Name.Name)
+func checkTooLongColumn(cols []string) error {
+	for _, col := range cols {
+		if len(col) > mysql.MaxColumnNameLength {
+			return ErrTooLongIdent.Gen("too long column %s", model.NewCIStr(col))
```

Comment:
Why wrap col with CIStr before convert it to string?  You can print col directly.
See: https://github.com/pingcap/tidb/blob/master/model/model.go#L295